### PR TITLE
added js conf and tech unite logo

### LINKED ISSUE #454: [BUG] Fix JS Conf Chile and Tech Unite Africa images
For the Angular, React and Vue homepages, the JS Conf Chile and Tech Unite Africa images are blurry under the Communities section.

Screen Shot 2022-11-09 at 3 05 14 PM

We need to replace those blurry images with higher quality ones.

### PR DESCRIPTION
**Pull request by aksy2512**
This fix resolves #454 
The changes follow the [contributing guidelines](https://github.com/thisdot/framework.dev/blob/main/CONTRIBUTING.md)
I have verified the fix works and introduces no further errors

Screenshot of the change - 
image

### CODE REVIEW

File: packages/site/src/data/angular/communities.ts [hunk 1]
```diff
@@ -204,7 +204,8 @@ export const communities: Community<typeof communityTags[number]>[] = [
...
...W7tPZpN6WZTs9xTalcJydqqlrnwQlzGgnOOhriPiMJ7Z3BYc9l8ZOywNqYzVsD4L2eCTYX1PIGsA2JHCL61fMQaGgMtmWGbm2zbcFraLWXeyllCWGtGMbt7Xq5c/C5WpUs/S2fnRbFDWyQyCSF5ZINTmnTyHgI4joVt5Q5CU9S8yNJhlOlxjALHn5TmHh4wRfhuomj3LmB15qpz2fJji3snizi51uYJDK2FnC8m1vVm/KzHQTvoOwydxE1FLFM5tnPbdwGrOBINuK4PMue3VYA6iDj2zJWEfWDmkdfUuup4GxsayNoaxoDWtGoACwAVf7reKDNipmnsid9eBwAAtYDykuP1VwcCs/GRzFZZ1+S0vw0c2izU0U3crVERexKAREQBERAEREAREQBERAEREAREQBERAEB2a+LQRxoikFhZObo5a0R1zXPtoEsdi+3ftOvlGniK66HLTD3C4qmjie17XdDmhUesrl1skYeo85Xjy1djv3G6NeSLfxPdEo4wd6Lpn8AY0tZfje8DRyAqscfxuWrlMsx4mNb2kbfktH5nWei0ai34XAUcM7wWne9f8GM6kpawuqyWy2mpAI3jfacamk2kj8B2zvTo2WXKorFajCtHMqK6MYycXdFy0e6DQPAzpHRk8EkT9HOwOHWveXLnD2jukHibFK49TVSaLmPImHb0OXavQ29PIszGd0xgBbRxFzv5kwswcYYDd3PZVzV1L5XukleXyON3OdrJ/3wcC8UV7DYSjh1amte3W+30sjXKcpawiIrJgEREB6VMDo3uZI0te0kOa7WCOBeauvKzJGKsGdfe6gCzZAL3HA14+MOsdRqrGsm6qlJ3+I5n8xnZQnjzhq5HWKoYPKFLEJK9pbvTh38DbUpOHIiUWAVlXzUEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBZAJNgLk6AALkk6gBwlb+E4LUVJtTwueOF1rRDledA5NatDJHIiOlIlmIkqeAge1ReADrd3x5gOGni8dSwy+J3lsS1/x195shTc9RXv9jMQ/uj+lnrWVeCLif1uv/bHv9Sz/AKeO8ws8B5CiLiyLC2lE5Xd1PUMiL3tD8uJzZazKIi2GIREQBERAEREAREQBERAEREAREQBERAEREBhbuD/Dx+EiLCp8jMo6y/KT4NngheqIvBfVLmdHYgiIhB//2Q==",
```
Comment (jdwilkin4) [inline packages/site/src/data/angular/communities.ts:208]: This image url is to long. 
Let's try something shorter like this one
```suggestion
	        "https://pbs.twimg.com/profile_images/1548041354806059008/CfEL7qut_400x400.jpg",   
```

File: packages/site/src/data/react/communities.ts [hunk 2]
```diff
@@ -223,7 +223,8 @@ export const communities: Community<typeof communityTags[number]>[] = [
...
...W7tPZpN6WZTs9xTalcJydqqlrnwQlzGgnOOhriPiMJ7Z3BYc9l8ZOywNqYzVsD4L2eCTYX1PIGsA2JHCL61fMQaGgMtmWGbm2zbcFraLWXeyllCWGtGMbt7Xq5c/C5WpUs/S2fnRbFDWyQyCSF5ZINTmnTyHgI4joVt5Q5CU9S8yNJhlOlxjALHn5TmHh4wRfhuomj3LmB15qpz2fJji3snizi51uYJDK2FnC8m1vVm/KzHQTvoOwydxE1FLFM5tnPbdwGrOBINuK4PMue3VYA6iDj2zJWEfWDmkdfUuup4GxsayNoaxoDWtGoACwAVf7reKDNipmnsid9eBwAAtYDykuP1VwcCs/GRzFZZ1+S0vw0c2izU0U3crVERexKAREQBERAEREAREQBERAEREAREQBERAEB2a+LQRxoikFhZObo5a0R1zXPtoEsdi+3ftOvlGniK66HLTD3C4qmjie17XdDmhUesrl1skYeo85Xjy1djv3G6NeSLfxPdEo4wd6Lpn8AY0tZfje8DRyAqscfxuWrlMsx4mNb2kbfktH5nWei0ai34XAUcM7wWne9f8GM6kpawuqyWy2mpAI3jfacamk2kj8B2zvTo2WXKorFajCtHMqK6MYycXdFy0e6DQPAzpHRk8EkT9HOwOHWveXLnD2jukHibFK49TVSaLmPImHb0OXavQ29PIszGd0xgBbRxFzv5kwswcYYDd3PZVzV1L5XukleXyON3OdrJ/3wcC8UV7DYSjh1amte3W+30sjXKcpawiIrJgEREB6VMDo3uZI0te0kOa7WCOBeauvKzJGKsGdfe6gCzZAL3HA14+MOsdRqrGsm6qlJ3+I5n8xnZQnjzhq5HWKoYPKFLEJK9pbvTh38DbUpOHIiUWAVlXzUEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBZAJNgLk6AALkk6gBwlb+E4LUVJtTwueOF1rRDledA5NatDJHIiOlIlmIkqeAge1ReADrd3x5gOGni8dSwy+J3lsS1/x195shTc9RXv9jMQ/uj+lnrWVeCLif1uv/bHv9Sz/AKeO8ws8B5CiLiyLC2lE5Xd1PUMiL3tD8uJzZazKIi2GIREQBERAEREAREQBERAEREAREQBERAEREBhbuD/Dx+EiLCp8jMo6y/KT4NngheqIvBfVLmdHYgiIhB//2Q==",
```
Comment (jdwilkin4) [inline packages/site/src/data/react/communities.ts:227]: Let's use this short image url instead
```suggestion
	        "https://pbs.twimg.com/profile_images/1548041354806059008/CfEL7qut_400x400.jpg",   
```

File: packages/site/src/data/vue/communities.ts [hunk 3]
```diff
@@ -225,7 +225,8 @@ export const communities: Community<typeof communityTags[number]>[] = [
...
...W7tPZpN6WZTs9xTalcJydqqlrnwQlzGgnOOhriPiMJ7Z3BYc9l8ZOywNqYzVsD4L2eCTYX1PIGsA2JHCL61fMQaGgMtmWGbm2zbcFraLWXeyllCWGtGMbt7Xq5c/C5WpUs/S2fnRbFDWyQyCSF5ZINTmnTyHgI4joVt5Q5CU9S8yNJhlOlxjALHn5TmHh4wRfhuomj3LmB15qpz2fJji3snizi51uYJDK2FnC8m1vVm/KzHQTvoOwydxE1FLFM5tnPbdwGrOBINuK4PMue3VYA6iDj2zJWEfWDmkdfUuup4GxsayNoaxoDWtGoACwAVf7reKDNipmnsid9eBwAAtYDykuP1VwcCs/GRzFZZ1+S0vw0c2izU0U3crVERexKAREQBERAEREAREQBERAEREAREQBERAEB2a+LQRxoikFhZObo5a0R1zXPtoEsdi+3ftOvlGniK66HLTD3C4qmjie17XdDmhUesrl1skYeo85Xjy1djv3G6NeSLfxPdEo4wd6Lpn8AY0tZfje8DRyAqscfxuWrlMsx4mNb2kbfktH5nWei0ai34XAUcM7wWne9f8GM6kpawuqyWy2mpAI3jfacamk2kj8B2zvTo2WXKorFajCtHMqK6MYycXdFy0e6DQPAzpHRk8EkT9HOwOHWveXLnD2jukHibFK49TVSaLmPImHb0OXavQ29PIszGd0xgBbRxFzv5kwswcYYDd3PZVzV1L5XukleXyON3OdrJ/3wcC8UV7DYSjh1amte3W+30sjXKcpawiIrJgEREB6VMDo3uZI0te0kOa7WCOBeauvKzJGKsGdfe6gCzZAL3HA14+MOsdRqrGsm6qlJ3+I5n8xnZQnjzhq5HWKoYPKFLEJK9pbvTh38DbUpOHIiUWAVlXzUEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBZAJNgLk6AALkk6gBwlb+E4LUVJtTwueOF1rRDledA5NatDJHIiOlIlmIkqeAge1ReADrd3x5gOGni8dSwy+J3lsS1/x195shTc9RXv9jMQ/uj+lnrWVeCLif1uv/bHv9Sz/AKeO8ws8B5CiLiyLC2lE5Xd1PUMiL3tD8uJzZazKIi2GIREQBERAEREAREQBERAEREAREQBERAEREBhbuD/Dx+EiLCp8jMo6y/KT4NngheqIvBfVLmdHYgiIhB//2Q==",
```
Comment (jdwilkin4) [inline packages/site/src/data/vue/communities.ts:229]:
```suggestion
	        "https://pbs.twimg.com/profile_images/1548041354806059008/CfEL7qut_400x400.jpg",   
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by jdwilkin4** [comment]
@aksy2512 

Are you still interested in working on this?

**Comment by aksy2512** [comment]
@jdwilkin4 I am facing some error while rebasing pushing. Should I create a new Pull request?

**Comment by jdwilkin4** [comment]
@aksy2512 

In this situation, you don't have to make these changes locally because these are all minor content changes for the image. 
So you can use the suggestion tool and click on Commit suggestion

Screen Shot 2022-11-22 at 10 07 00 AM

**Comment by jdwilkin4** [comment]
@aksy2512 

Don't forget to address all of the review comments so we can merge your PR 👍

**Comment by aksy2512** [comment]
@jdwilkin4 How can I add the review comments now? Please help

**Comment by jdwilkin4** [comment]
@aksy2512 

It looks like you already made the necessary changes to one of my review comments because I can see this commit you made here
Screen Shot 2022-11-22 at 10 38 16 AM

But you should also see these comments here

Screen Shot 2022-11-22 at 10 41 19 AM

and here
Screen Shot 2022-11-22 at 10 41 37 AM

I need to click on the Commit suggestion buttons for those two remaining images and then we can merge your PR.

**Comment by jdwilkin4** [comment]
@aksy2512 

Thank you for making those changes.
As I was testing it, I discovered that the tech unit Africa image has its URL signature expire. 
I went ahead and pushed an update to replace that image with a new. 

We should be good now to merge this PR 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**